packages/storage-adapter-indexeddb/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ciscospark/storage-adapter-indexeddb",
```

Comment:
Can we rename this to `storage-adapter-local-forage`. I could imagine a much more robust indexeddb implementation at some point, so I think it makes sense to be a bit more explicit about the name here.